### ISSUE TITLE
docker build fails

### ISSUE THREAD

**Issue by bignay2000** [microsoft/sbom-tool#747]
Unable to docker build v.2.2.9  or v.2.2.8.   

```
admin@vmdecld01 /srv/dockersource/sbom-tool $ docker build . -t ms_sbom_tool
DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
            Install the buildx component to build images with BuildKit:
            https://docs.docker.com/go/buildx/

Sending build context to Docker daemon  4.165MB
Step 1/12 : FROM mcr.microsoft.com/dotnet/sdk AS build-env
 ---> 3d4b35ceef07
Step 2/12 : COPY . /app
 ---> 9bc82bd0d10d
Step 3/12 : WORKDIR /app/src/Microsoft.Sbom.Tool
 ---> Running in 7528e76c69a3
Removing intermediate container 7528e76c69a3
 ---> 5d3a11ee097a
Step 4/12 : ARG RUNTIME=linux-x64
 ---> Running in d32ea4ef6ea3
Removing intermediate container d32ea4ef6ea3
 ---> 30bf3fe4ebf9
Step 5/12 : RUN dotnet publish -f net8.0 -r $RUNTIME --self-contained true -p:DebugType=None -p:DebugSymbols=false -p:PublishSingleFile=true -p:IncludeAllContentForSelfExtract=true -o output
 ---> Running in 8ee4b9c6d41d
  Determining projects to restore...
  Restored /app/src/Microsoft.Sbom.Contracts/Microsoft.Sbom.Contracts.csproj (in 2.14 min).
  Restored /app/src/Microsoft.Sbom.Adapters/Microsoft.Sbom.Adapters.csproj (in 2.4 min).
  Restored /app/src/Microsoft.Sbom.Common/Microsoft.Sbom.Common.csproj (in 19.27 sec).
  Restored /app/src/Microsoft.Sbom.Api/Microsoft.Sbom.Api.csproj (in 21.94 sec).
  Restored /app/src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Microsoft.Sbom.Parsers.Spdx22SbomParser.csproj (in 12 ms).
/app/src/Microsoft.Sbom.Extensions/Microsoft.Sbom.Extensions.csproj : error NU1903: Warning As Error: Package 'System.Text.Json' 8.0.4 has a known high severity vulnerability, https://github.com/advisories/GHSA-8g4q-xg66-9fp4 [/app/src/Microsoft.Sbom.Tool/Microsoft.Sbom.Tool.csproj]
  Failed to restore /app/src/Microsoft.Sbom.Extensions/Microsoft.Sbom.Extensions.csproj (in 18 ms).
  Restored /app/src/Microsoft.Sbom.Tool/Microsoft.Sbom.Tool.csproj (in 18.73 sec).
  Restored /app/src/Microsoft.Sbom.Extensions.DependencyInjection/Microsoft.Sbom.Extensions.DependencyInjection.csproj (in 89 ms).
The command '/bin/sh -c dotnet publish -f net8.0 -r $RUNTIME --self-contained true -p:DebugType=None -p:DebugSymbols=false -p:PublishSingleFile=true -p:IncludeAllContentForSelfExtract=true -o output' returned a non-zero code: 1
```
```
admin@vmdecld01 /srv/dockersource/sbom-tool $ cat .git/HEAD
08ba73d303228eb4d92a6a5f75350d78230bca30
admin@vmdecld01 /srv/dockersource/sbom-tool $ git describe --exact-match --tags
v2.2.8
```

**Comment by bignay2000** [comment]
Can you let me know a version that does a docker build successfully

**Comment by kpb** [comment]
I had this problem earlier, but it seems to be fixed in the main branch.
```
$ cat .git/HEAD 
ref: refs/heads/main
```

**Comment by bignay2000** [comment]
Deleted my checkout of this repo and did a clean git clone.  

I was successful in building the container from the latest master.   

So hopefully we can get a new release that contains this fix from the master branch.

**Comment by DaveTryon** [comment]
@bignay2000, we aren't planning on more 2.x releases, but this is fixed in the 3.0.1 release.

**Comment by bignay2000** [comment]
Any plans to put the docker image release hosted on hub.docker.com?
